### PR TITLE
Small Refactoring - IMapObject: Tree drawing

### PR DESCRIPTION
**Pull request by homoroselaps**
move constants away from an interface to the only point there are used

### CODE REVIEW

File: jsettlers.graphics/src/jsettlers/graphics/map/draw/MapObjectDrawer.java [hunk 1]
```diff
@@ -71,6 +72,10 @@
 	private static final int[] TREE_CHANGING_SEQUENCES = new int[] {
 			3, 3, 6, 9, 9, 18, 18,
 	};
+	static final float TREE_CUT_1 = 0.03F;
+	static final float TREE_CUT_2 = 0.06F;
+	static final float TREE_CUT_3 = 0.09F;
+	static final float TREE_TAKEN = 0.1F;
```
Comment (michaelzangl) [inline jsettlers.graphics/src/jsettlers/graphics/map/draw/MapObjectDrawer.java]: Why is this not used by logic?

**Comment by michaelzangl** [comment]
Please work based on this branch for graphics improvements:
https://github.com/jsettlers/settlers-remake/pull/325

Feel free to add your thoughts on the changes there.

File: jsettlers.graphics/src/jsettlers/graphics/map/draw/MapObjectDrawer.java (hunk 1 above)
Comment (michaelzangl) [inline jsettlers.graphics/src/jsettlers/graphics/map/draw/MapObjectDrawer.java]: I had a look at it. Logic really does not change this and is not actively in sync with the tree cutting process.

So this is good to be moved here.

**Comment by andreas-eberle** [comment]
Can you update the branch, so I can merge it?

**Comment by andreas-eberle** [comment]
I will close this PR in favor of #355 which integrates this change.

**Pull request closed without merging**